examples/nodejs/src/experimental-features/metadata.ts

```diff
@@ -0,0 +1,8 @@
+import { Address } from "@marlowe.io/language-core-v1";
```

Comment:
I don't understand the link between metadata and splitting an address in 2

Reply:
https://github.com/input-output-hk/marlowe-ts-sdk/pull/136#discussion_r1447462610